sensors/listener.go

```diff
@@ -102,9 +103,14 @@ func (sensorCtx *SensorContext) ListenEvents(ctx context.Context, stopCh <-chan
 			for _, t := range triggers {
 				triggerNames = append(triggerNames, t.Template.Name)
 			}
-			conn, err := ebDriver.Connect()
+			var conn eventbusdriver.Connection
+			err = sources.Connect(&common.DefaultRetry, func() error {
+				var err error
+				conn, err = ebDriver.Connect()
+				return err
+			})
 			if err != nil {
-				logger.Error("failed to connect to event bus", zap.Error(err))
+				logger.Fatal("failed to connect to event bus", zap.Error(err))
```

Comment:
No need to change to `fatal`, the returned error will be wrapped as a `fatal` error in main.go